crates/ra_cfg/src/lib.rs

```diff
@@ -53,4 +53,13 @@ impl CfgOptions {
     pub fn insert_features(&mut self, iter: impl IntoIterator<Item = SmolStr>) {
         iter.into_iter().for_each(|feat| self.insert_key_value("feature".into(), feat));
     }
+
+    /// Shortcut to set cfgs
+    pub fn insert_cfgs(&mut self, iter: impl IntoIterator<Item = SmolStr>) {
+        iter.into_iter().for_each(|cfg| match cfg.find('=') {
+            Some(split) => self
+                .insert_key_value(cfg[0..split].into(), cfg[split + 1..].trim_matches('"').into()),
```

Comment:
In theory, we should unescape the RHS here, but in practice, I don't think it is relevant here, right?